bb-new.html

```diff
@@ -0,0 +1,282 @@
+                    <a href="#transporation">Transportation</a>
+                    <a href="#consumer">Consumer industries</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#blog">Blog</a>
+                  <div class="dropdown-content">
+                    <a href="#enterprise">Enterprise Blog</a>
+                    <a href="#developer">Developer</a></li>
+                    <a href="#support">Support Blog</a>
+                  </div>
+              </div>
+              <div class="topnav-right">
+                <img src="/Users/berangerelam/Desktop/images-blackberry/search.png">
+              </div>
+              <div class="dropdown topnav-right">
+                <a href="#contact">Telephone icon</a>
+                  <div class="dropdown-content">
+                    <a href="#register">Register for updates</a>
+                    <a href="#sales">Contact Sales</a></li>
+                    <a href="#contactus">Contact Us</a>
+                  </div>
+                </div>            
+            </div>
+          </header>         
+        </div>         
+      </div>
+      <div class="row">
+        <div class="column"> 
+            <img src="/Users/berangerelam/Desktop/images-blackberry/windmill.jpg" alt="several windmills in a line">
+            <h1 class="hero-text">OUR DEVICES CHANGED WHERE WE WORK. </h1>
+            <h1 class="hero-text">OUR SECRUITY IS CHANGING HOW WE WORK.</h1>
+            <p class="hero-text">From cybersecurity, to critical event management, to the internet of Things, when we say intelligent secuirty everywhere- we mean everywhere</p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column">
+          <div class="btn">
+            <button>WATCH VIDEO</button>
+          </div>        
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-2">
+          <h2>SECURITY SHOULDN’T JUST BE SMART. IT SHOULD BE INTELLIGENT.</h2>
+          <br/>
+          <p>We’re known for secure handsets that changed the world, and how we work. Today, we’re reimagining protection, again, with artificial intelligence and machine learning. We make everything more secure—and you, more productive.</p> 
+
+          <a href="#cybersecurity">CYBERSECURITY</a>
+          <p>Our cybersecurity solutions, Cyber Suite and Spark, protect your work and your ability to work everywhere. </p>
+          <button>LEARN MORE</button>
+          <br/>
+          <br/>
+          <a href="#">CRITICAL EVENT MANAGEMENT</a>
+          <br/>
+          <a href="#">INTERNET OF THINGS (loT)</a>
+        </div>
+        <div class="col-2">
+          <img src="/Users/berangerelam/Desktop/images-blackberry/man-outside-laptop.jpg" alt="man outside on laptop">
+        </div>
+      </div>    
+      </div>      
+      <div class="row">
+        <div class="col-2" style="background-color: blue;">         
```

Comment:
not sure if you meant to leave this in, just so you could see things laid out on screen
but in general, you don't directly style things this way, unless you do so with Javascript

Reply:
How do we target this div? I don’t remember how to do this with two classes.

Reply:
There is only one class here and you would not directly style col-2, as it's a class that is used in many locations.